client/gutenberg/extensions/map/settings.js

```diff
@@ -0,0 +1,96 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+
+export const settings = {
+	name: 'jetpack/map',
+	title: __( 'Map', 'jetpack' ),
+	icon: (
+		<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+			<path d="M20.5 3l-.16.03L15 5.1 9 3 3.36 4.9c-.21.07-.36.25-.36.48V20.5c0 .28.22.5.5.5l.16-.03L9 18.9l6 2.1 5.64-1.9c.21-.07.36-.25.36-.48V3.5c0-.28-.22-.5-.5-.5zM15 19l-6-2.11V5l6 2.11V19z" />
+			<path d="M0 0h24v24H0z" fill="none" />
+		</svg>
+	),
+	category: 'jetpack',
+	keywords: [ __( 'map', 'jetpack' ), __( 'location', 'jetpack' ), 'jetpack' ],
```

Comment:
I think we can drop `jetpack` keyword since the category is already used as a search word.

Reply:
I didn't have it initially. In the first PR I believe there was a comment asking me to add `jetpack` as a category. Did this thinking change recently?

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/28498/commits/28f78a009ac78fa9eff61689f8fa60636a4761c7